adserver/templates/adserver/base.html

```diff
@@ -90,9 +90,19 @@ <h6 class="text-muted">{{ publisher }}</h6>
                 </a>
               </li>
 
+              {% if request.user.is_staff %}
```

Comment:
```suggestion
              {% if request.user.is_staff and publisher.record_placements %}
```

Since the `record_placements` is always present in the settings form, should we just make this available if the publisher has it turned on?

Reply:
Yea, is_staff is leftover from the old PR that didn't have the option, we can remove it.